File: exercises/D502/solution.py
file = open('weather.txt', 'r')
max_temp = 0
min_temp = 100
for line in file:
    words = line[32:].split()
    x = int(words[0])
    max_temp = max(max_temp, x)
    min_temp = min(min_temp, x)
file.close()
print(min_temp, max_temp)
    

File: exercises/D502/solution.cpp
#include <fstream>
#include <iostream>
using namespace std;
int main() {
    ifstream fin("weather.txt");
    int max_temp = 0, min_temp = 100;
    while (!fin.eof()) {
        fin.ignore(32);
        int x;
        string dummy;
        fin >> x;
        getline(fin, dummy);
        max_temp = max(x, max_temp);
        min_temp = min(x, min_temp);
    }
    fin.close();
    cout << min_temp << " " << max_temp << endl;
}
